src/content/en/_shared/contributors/mco.html

```diff
@@ -0,0 +1,24 @@
+<style>
```

Comment:
You don't have to create this, it's automatically created based on the `src/data/_contributors.yaml` file. 

Add yourself to that file, then run gulp build, it'll automatically generate this file for you.